components/eamxx/cime_config/namelist_defaults_scream.xml

```diff
@@ -274,19 +274,25 @@ be lost if SCREAM_HACK_XML is not enabled.
 
     <!-- Simple Prescribed Aerosols (SPA) -->
     <spa inherit="atm_proc_base">
-      <spa_remap_file type="file">UNSET</spa_remap_file>
+      <spa_remap_file type="file"
+        doc="File containing mapping data from the grid of spa_data_file to the model grid. Unused if the grid is the same.">UNSET</spa_remap_file>
       <spa_remap_file hgrid="ne4np4">${DIN_LOC_ROOT}/atm/scream/maps/map_ne30_to_ne4_mono_20220502.nc</spa_remap_file>
-      <spa_remap_file hgrid="ne4np4.pg2">${DIN_LOC_ROOT}/atm/scream/maps/map_ne30np4_to_ne4pg2_mono.20220714.nc</spa_remap_file>
+      <spa_remap_file hgrid="ne4np4.pg2">none</spa_remap_file>
       <spa_remap_file hgrid="ne30np4">none</spa_remap_file>
-      <spa_remap_file hgrid="ne30np4.pg2">${DIN_LOC_ROOT}/atm/scream/maps/map_ne30np4_to_ne30pg2_mono.20220714.nc</spa_remap_file>
+      <spa_remap_file hgrid="ne30np4.pg2">${DIN_LOC_ROOT}/atm/scream/maps/map_ne4pg2_to_ne30pg2_20231201.nc</spa_remap_file>
       <spa_remap_file hgrid="ne120np4">${DIN_LOC_ROOT}/atm/scream/maps/map_ne30np4_to_ne120np4_mono_20220502.nc</spa_remap_file>
       <spa_remap_file hgrid="ne512np4">${DIN_LOC_ROOT}/atm/scream/maps/map_ne30np4_to_ne512np4_mono_20220506.nc</spa_remap_file>
       <spa_remap_file hgrid="ne120np4.pg2">${DIN_LOC_ROOT}/atm/scream/maps/map_ne30np4_to_ne120pg2_intbilin_20221012.nc</spa_remap_file>
       <spa_remap_file hgrid="ne256np4.pg2">${DIN_LOC_ROOT}/atm/scream/maps/map_ne30np4_to_ne256pg2_intbilin_20221011.nc</spa_remap_file>
       <spa_remap_file hgrid="ne512np4.pg2">${DIN_LOC_ROOT}/atm/scream/maps/map_ne30np4_to_ne512pg2_intbilin_20221012.nc</spa_remap_file>
       <spa_remap_file hgrid="ne1024np4.pg2">${DIN_LOC_ROOT}/atm/scream/maps/map_ne30np4_to_ne1024pg2_intbilin_20221012.nc</spa_remap_file>
 
-      <spa_data_file type="file">${DIN_LOC_ROOT}/atm/scream/init/spa_file_unified_and_complete_ne30_20220428.nc</spa_data_file>
+      <spa_data_file type="file"
+        doc="File containing aerosol data. Must be on same grid as the atm, or a coarser one"
+        >${DIN_LOC_ROOT}/atm/scream/init/spa_file_unified_and_complete_ne30_20220428.nc</spa_data_file>
+      <spa_data_file hgrid="ne30np4.pg2" type="file">${DIN_LOC_ROOT}/atm/scream/init/spa_file_unified_and_complete_ne4pg2_20231222.nc</spa_data_file>
```

Comment:
We should probably remap the ne30np4 data to ne30pg2 and use it here.  ne4pg2 is significantly smoothed compared to ne30np4.

Reply:
Yeah, I was trying to trigger a remap in test though. Perhaps I can use a testmod, to force a nightly test to use ne4pg2 data, and remap...